qcoverage/src/org/qcmg/coverage/Coverage.java

```diff
@@ -97,7 +95,7 @@ private void writeXMLCoverageReport(final QCoverageStats report) throws Exceptio
 		m.setProperty(Marshaller.JAXB_ENCODING, "UTF-8");
 		m.setProperty(Marshaller.JAXB_FORMATTED_OUTPUT, Boolean.TRUE);
 		m.marshal(report, sw);
-		final File file = new File(options.getOutputFileNames()[0]);
+		final File file = new File(options.getOutputFileNames()[0]+ ".xml");
```

Comment:
what if the output file name already has a ".xml" extension?

Reply:
it will be *.xml.xml.  eg. "--output-format TXT --output-format XML --output  /dir/output". There will be two output file:
/dir/output.xml and /dir/output.xml

Reply:
I think adding code to make this smarter is a good idea

Reply:
function added. However, our WDL pipeline has to be smart as well to predict the output file name.